.travis.yml

```diff
@@ -1,3 +1,24 @@
 language: node_js
 node_js:
-  - "node"
\ No newline at end of file
+  - "node"
+  - "10"
+  - "9"
```

Comment:
👍

Reply:
v5 + v6 continue to fail, v5 started to fail once the below fixed had been applied incl. reactivating the priorly disabled test suites.
I will remove both versions incl. also 7 from the travis build.